app/services/unblock_brevo_transactionnal_contact.rb

```diff
@@ -0,0 +1,19 @@
+# https://developers.brevo.com/reference/delete_smtp-blockedcontacts-email
+class UnblockBrevoTransactionnalContact
```

Comment:
je pense que c’est une mini faute d’orthographe en anglais un seul `n` à transactional : https://www.merriam-webster.com/dictionary/transactional

```suggestion
class UnblockBrevoTransactionalContact
```

Est-ce que ça débloque vraiment que côté transactionnel et pas marketing ? 

sinon peut-être que `UnblockBrevoContact` voire `UnblockBrevoEmail` pour simplifier ?

Reply:
Oups, bien vu pour typo

Effectivement, c’est spécifique à la partie transactionnel. Je préfère donc que ça reste comme ça dans le cas où un jour on imagine que ce code le fasse pour la partie marketing